xplique/commons/operators.py

```diff
@@ -111,6 +137,110 @@ def segmentation_operator(model: Callable,
     scores = tf.reduce_sum(model(inputs) * targets, axis=(1, 2, 3))
     return scores
 
+class Tasks(Enum):
+    """
+    Enumeration of different tasks for which we have defined operators
+    """
+    CLASSIFICATION = auto()
+    REGRESSION = auto()
+    BINARY_SEGMENTATION = auto()
+    SEGMENTATION = auto()
+
+enum_to_method = {
```

Comment:
same here, you don't need to define 2 object, the goal of enum is to do the mapping (instead of auto put the function to call), see what I did here: 
https://github.com/deel-ai/xplique/blob/master/xplique/attributions/global_sensitivity_analysis/gsa_attribution_method.py#L17

Reply:
I changed it :)